content/blog/global-case-study.md

```diff
@@ -16,76 +16,82 @@ A leading comprehensive global mental health solution for employers. It started
+**Outcome**: Users can access content in their preferred language.
 
 #### Translating Emails
 
-As is prevalent, we also utilize mails for communication with the user and as we went global, we had to make sure that our mails are also translated.
+**Problem**: Client needed an easy way to localize emails and update email content
+
+**Solution**:
 
-Initially all of our mails were handled by rails views, and every mail was basically an embedded ruby file that plugged in data at required segments and sent out the mail.
+- Introduce third-party mail service provider Iterable
+- Migrate existing code heavy mailers to easy to edit Iterable templates
+- Integrate Iterable with existing mail sending behaviour
 
-This posed some challenges, the content of our mails changes often and relying on devs to make these changes frequently was turning out to be a bottleneck for our marketing efforts.
+**Outcome**: Mails content is now translated and content can be tweaked without any dependency on developer solving a major bottleneck for client's marketing team.
 
-We needed a tool that someone from a non-tech background could easily use, while still being powerful enough to provide the features present in embedded ruby mailers. That is when we introduced Iterable, a service that allowed us to do exactly these things, it provided an easy-to-use WYSIWYG drag and drop editor to create mail templates and also had embedded ruby like data plugin features using its robust API, along with these it also came with detailed metrics and telemetry service, campaigns and workflows.
+### Supporting Right-to-Left Languages
 
-We just had to setup the mail templates on Iterable and setup the API with our backend application to send in required data, and boom! You had globalized emails!
+**Problem**: UI needed to update based on language direction
 
-This solved a major bottleneck for marketing too as they no longer needed to rely on developers to make changes to the mail content.
+**Solution**:
 
-#### Locales
+- Identifying elements and interactions that needed updates
+- Update components with logical CSS properties
+- Add support for bidirectional text and icons
+- Add smart exceptions based on content type
 
-The length of words and sentences can wary in different locales. This causes layout issues in design of the User Interface which was initially built with `en` in mind. The solution to this in most cases, is a simple fix to the design elements property to handle it dynamically.
+**Outcome**: Users preferring RTL languages could naturally interact with the application.
 
-Now, as we kept on expanding to more countries supporting more locales, we reached a locale that uses a script that is read and written in right-to-left direction as opposed to `en` using Latin script that goes in left-to-right direction. Several factors have to be taken into consideration such as:
+### Conclusion
```

Comment:
In the conclusion we can add the funding related thing, and then two sentences about how crowdin helped.